setup.py

```diff
@@ -22,7 +22,8 @@
     name='quantum_computing_project',
     use_scm_version=True,  # tags the project version from the git tag
     setup_requires=['setuptools_scm', 'pytest-runner'],
-    packages=setuptools.find_packages(exclude=['docs', 'tests']),
+    packages=setuptools.find_packages(where="qcp", exclude=['docs', 'tests']),
+    package_dir={"": "qcp"},
```

Comment:
Changes were required to get the test code to discover python modules correctly.